lib/shoryuken/fetcher.rb

```diff
@@ -9,20 +9,40 @@ def initialize(group)
     end
 
     def fetch(queue, limit)
-      started_at = Time.now
+      do_with_retry(10) do
```

Comment:
Any thoughts on making this configurable?

Reply:
@ethangunderson I usually make that stuff configurable on demand basis, if someone asks to configure, just to avoid adding extra options.